general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -192,6 +191,32 @@ def get_job(self, job_id: str) -> dict[str, str]:
         """
         return self.fetch_jobs([job_id])[job_id]
 
+    def cancel_jobs(
+        self,
+        job_ids: list[str],
+        **kwargs: Any,
+    ) -> dict[str, list[str] | dict[str, str]] | str:
+        """Cancel jobs associated with given job ids.
+
+        Args:
+            job_ids: The UUID of the job (returned when the job was created).
+            kwargs:  Extra options needed to fetch jobs.
+                - cq_token: CQ Cloud credentials.
+
+        Returns:
+            A dictionary mapping the job id to the cancellation attempt result or an error message.
```

Comment:
might need an update: by the time we get here i think all that's left in the dict is `{"succeeded": [<ids>]}`

(also might be reasonable to just return the list of ids, instead of the full dictionary)